app/Keycloak/Repositories/EloquentKeycloakClientRepository.php

```diff
@@ -24,13 +25,12 @@ public function save(Client ...$clients): void
                 KeycloakClientModel::query()
                     ->updateOrCreate(
                         [
-                            'client_id' => $client->clientId,
+                            'client_secret' => $client->clientSecret, // secret could actually change because of secret rotation
```

Comment:
Is this something we want? What happens to an existing integration, will it stop working because the secret is different?

Reply:
I also think this code is wrong. The first array is used to find the entity and the second array is used to update the values. See https://laravel.com/docs/11.x/eloquent#updates

Reply:
Respectfully, I don't think so.

> 
> In the example below, if a flight exists with a departure location of Oakland and a destination location of San Diego, its price and discounted columns will be updated. If no such flight exists, a new flight will be created which has the attributes resulting from merging the first argument array with the second argument array:
> 
> $flight = Flight::updateOrCreate(
>     ['departure' => 'Oakland', 'destination' => 'San Diego'],
>     ['price' => 99, 'discounted' => 1]
> );
> 

If you look at the auth0 repo you can also see the same structure
```

Auth0ClientModel::query()
                    ->updateOrCreate(
                        [
                            'auth0_client_id' => $auth0Client->clientId,
                            'auth0_tenant' => $tenant,
                        ],
                        [
                            'id' => $auth0Client->id->toString(),
                            'integration_id' => $auth0Client->integrationId->toString(),
                            'auth0_client_id' => $auth0Client->clientId,
                            'auth0_client_secret' => $auth0Client->clientSecret,
                            'auth0_tenant' => $tenant,
                        ]
                    );
```

Reply:
```
'client_secret' => $client->clientSecret, // secret could actually change because of secret rotation
```

If the secret is different, this would mean that the client will not be found and a new one would be created.